wagl/acquisition/landsat.py

```diff
@@ -342,11 +342,99 @@ def radiance_data(self, window=None, out_no_data=-999, esun=None):
         return radiance
 
 
+class Landsat9Acquisition(LandsatAcquisition):
+
+    """Landsat 9 acquisition."""
+
+    def __init__(
+        self,
+        pathname,
+        uri,
+        acquisition_datetime,
+        band_name="BAND 1",
+        band_id="1",
+        metadata=None,
+    ):
+        super(Landsat9Acquisition, self).__init__(
+            pathname,
+            uri,
+            acquisition_datetime,
+            band_name=band_name,
+            band_id=band_id,
+            metadata=metadata,
+        )
+
+        self.platform_id = "LANDSAT_9"
+
+        # TODO should it be OLI-2? maybe technically but would probably mess up naming
+        self.sensor_id = "OLI"
+
+        # TODO not sure about this, just copied from Landsat-8
+        # the downloaded TLE files are in /g/data/v10/eoancillarydata-2/sensor-specific
+        # they look pretty similar to Landsat-8 to me
+        self.tle_format = "L9%4d%sASNNOR.S00"
+
+        self.tag = "LS9"
+
+        # TODO I don't know what this means or how to get it
+        # the value is from Landsat-8
+        self.altitude = 705000.0
```

Comment:
These values are used as fallback values for when there is no TLE information.

In the past I've pulled a static snapshot from n2yo.com (which you've already posted), or from Wikipedia.  I wouldn't worry too much if they're too different as (whilst I haven't actually tested) I'm pretty confident the impact would be fairly insignificant.